pkg/analyzer/java/parse_manifests.go

```diff
@@ -83,7 +76,7 @@ func parseManifestMF(locationSoFar string, f *zip.File) (parsedManifestMF, error
 			isTemplatedManifestFile = true
 			continue
 		}
-
+		valueFromLine = strings.TrimSpace(valueFromLine)
```

Comment:
should we do this before the `if valueFromLine == ""` check?

Reply:
Ah yeah, good call